### PR TITLE
Feature route53 module

### PR DESCRIPTION
**Pull request by avmikholap**

### CODE REVIEW

File: terraform/modules/aws/route53/main.tf
Comment (ggramal) [inline terraform/modules/aws/route53/main.tf:24]: can we make change this to create multiple CNAME records?

File: terraform/modules/aws/route53/main.tf
Comment (ggramal) [inline terraform/modules/aws/route53/main.tf:18]: hmm seems this is a hardcoded name, can we use variables here to set it :thinking:

Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:18]: Will update here as soon as the alb module is available. That’s why I immediately left a comment about why I did this now.

File: terraform/modules/aws/route53/main.tf
Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:24]: I think we can try to change it

File: terraform/modules/aws/route53/main.tf
Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:18]: yeah but the question here will you update it also using hardcode?

Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:18]: No, we will not use hardcode. We will try to do it right, as the mentor taught)).

File: terraform/modules/aws/route53/main.tf
Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:24]: @ggramal  made it possible to create multiple CNAMEs. I checked - everything works.

File: terraform/modules/aws/route53/main.tf
Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:18]: @ggramal Can we skip this now if we fix it in the next PR, when there will be an ALB module?

Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:18]: ok lets skip

File: terraform/modules/aws/route53/main.tf
Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:24]:
```
$ terraform plan
╷
│ Error: Missing newline after argument
│ 
│   on dns_viriables.tf line 7, in variable "records":
│    7:   description = "A string of DNS records, example: ["192.158.1.1"]"
│ 
│ An argument definition must end with a newline.
╵
```
:thinking:

Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:24]: oh sry my bad its wrong team

Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:24]:
```
Initializing the backend...
Initializing modules...
- route53 in ../../../modules/aws/route53
- vpcs in 
╷
│ Error: Unreadable module directory
│ 
│ Unable to evaluate directory symlink: lstat ../../../modules/aws/vpc: no such file or directory
╵

╷
│ Error: Unreadable module directory
│ 
│ The directory  could not be read for module "vpcs" at main.tf:36.
╵
```

Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:24]: maybe you need to merge master :thinking:

Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:24]: Yes, you need to merge this branch into master. I checked it locally - 2 modules completed the terraform plan.
What should we do? We cannot merge until we receive approval. We can push the vpc module into the current brunch so that you can make a terraform plan.

Comment (ggramal) [reply terraform/modules/aws/route53/main.tf:24]: so yeah vpc module is in master branch so its in production state. What you need to do is to merge master into this branch

Comment (avmikholap) [reply terraform/modules/aws/route53/main.tf:24]: @ggramal Corrected, check.

**Review by ggramal — APPROVED** [review]

**Pull request merged**
